File: v2/testdata/java/src/main/java/com/example/services/CurrencyUtils.java
package com.example.services;

import java.text.NumberFormat;
import java.util.Currency;
import java.util.List;

public class CurrencyUtils {

    public static double calculateTotal(List<Double> items, double taxRate) {
        double total = 0.0;
        for (double item : items) {
            total += item;
        }
        return total * (1 + taxRate);
    }

    public static String formatCurrency(double amount, String currencyCode) {
        Currency currency = Currency.getInstance(currencyCode);
        NumberFormat format = NumberFormat.getCurrencyInstance();
        format.setCurrency(currency);
        return format.format(amount);
    }
}

File: testdata/csharp/src/Services/CurrencyUtils.cs
using System;
using System.Globalization;

namespace MyApp.Services;

public static class CurrencyUtils
{
    public static double CalculateTotal(double[] items, double taxRate)
    {
        double total = 0.0;
        foreach (var item in items)
            total += item;
        return total * (1 + taxRate);
    }

    public static string FormatAmount(double amount, string currencyCode)
    {
        var culture = new CultureInfo("en-US");
        return amount.ToString("C", culture) + " " + currencyCode;
    }
}
